README.md

```diff
@@ -162,6 +162,7 @@
         + [Stylus](http://learnboost.github.io/stylus/): Expressive, dynamic, robust CSS
         + [TypeScript](http://www.typescriptlang.org/) is a typed superset of JavaScript that compiles to plain JavaScript (by Microsoft)
         + [CoffeeScript](http://coffeescript.org/) is a little language that compiles into JavaScript.
+        + [Babel](https://babeljs.io/) is a compilter to generate next generation JavaScript, today.
```

Comment:
> Babel trans-compiles JSX to JS

Agreed, but its main focus is compiling ES2015 to ES5. Maybe it just deserves a place in both sections.